README.md

```diff
@@ -80,20 +83,20 @@ After build Inclavare Containers on your system, you can use the following comma
 sudo make install
 ```
 
-`rune` will be installed to `/usr/local/bin/rune` on your system. `shim-rune` will be installed to `/usr/local/bin/containerd-shim-rune-v2`. `sgx-tools` will be installed to `/usr/local/bin/sgx-tools`.
+`rune` will be installed to `/usr/local/bin/rune` on your system. `shim-rune` will be installed to `/usr/local/bin/containerd-shim-rune-v2`. `epm` will be installed to `/usr/local/bin/epm`. `sgx-tools` will be installed to `/usr/local/bin/sgx-tools`.
```

Comment:
In [epm/dist/rpm/epm.spec](https://github.com/alibaba/inclavare-containers/blob/master/epm/dist/rpm/epm.spec) and [epm/dist/deb/debian/rules](https://github.com/alibaba/inclavare-containers/blob/master/epm/dist/deb/debian/rules), epm will be installed to `/usr/bin/epm` ,  so you also need to revise `epm/dist/deb` and `epm/dist/rpm` files.

Reply:
I add a new issue #285